shawn_baugh_2/jsfun1/script.js

```diff
@@ -0,0 +1,40 @@
+var x = [3,5,'Dojo','rocks','Michael','Sensei']
+for(var i = 0;  i < x.length; i++){
+    console.log(x[i])
+}
+x.push(100);
+x.push(['hello','world','javascript is fun'])
+console.log(x)
+for(var i = 0; i < 501; i++){
+    i = i + i
+}
+console.log(i)
+// minimum
+function minimum() {
+    var y = [1, 5, 90, 25, -3, 0]
+    var min = y[0]
+    for(var i = 1; i < y.length; i++){
+        if(min > y[i]){
+            min = y[i]
+        }
+    } console.log(min)
+}
+minimum()
+// average
+function avg() {
+    var y = [1, 5, 90, 25, -3, 0]
+    for(var i = 1; i < y.length; i++){
+        i = i + i
+    } console.log(i/y.length)
+}
+avg()
```

Comment:
your average function does not work.  Try and rewrite it.  Make it so array gets passed into the function instead of always being `[1,5,90,25,-3,0]`.  Think about steps you need to take to calculate an average.  Sum / Length.